pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1899,6 +1899,15 @@ func handleDelete(ctx *domainContext, key string, status *types.DomainStatus) {
 	if status.Activated {
 		doInactivate(ctx, status)
 	} else {
+		if status.IsContainer {
```

Comment:
Is this good enough? Isn't status.Activated still set for an unexpectedly exiting container?

Reply:
Yes, it is good enough. If an instance is activated then it will go through doInactivate flow and we are removing apps after stopping them from the rkt.